src/pyedb/configuration/cfg_ports_sources.py

```diff
@@ -30,15 +30,15 @@ def pedb(self):
         return self._pdata.pedb
 
     @pyedb_function_handler
-    def __init__(self, pdata, **kwargs):
```

Comment:
The purpose of **kwargs is to make the input args expandable. Is there a strong reason to change this?

Reply:
I got some trouble while implementing. I found it at this harder to read since at the end we know what we are passing. I can try reverting.

Reply:
Please revert. Even with dict, we still don't know what is inside.